### ISSUE TITLE
htmldf todo

### ISSUE THREAD

**Issue by alastairrushworth** [alastairrushworth/htmldf#1]
- [x] add code column using text classifier
    - [x] simple test set for future improvements
    - [x] add training script to repo and to .Rbuildignore
    - [x] test for r / python code inference
    - [x] check correct behaviour when no code tags
    - [x] code tags types up to date?
- [x] train code text classifier: R, Python
- [x] datetime accessed
- [x] social column of tibbles : add other sites
- [x] loop once only over rows
- [x] add webpage publication date
